src/lime/_internal/backend/html5/HTML5Window.hx

```diff
@@ -839,6 +842,7 @@ class HTML5Window
 
 							if (touch == primaryTouch)
 							{
+								parent.clickCount = event.detail;
```

Comment:
[According to the documentation](https://developer.mozilla.org/en-US/docs/Web/API/UIEvent/detail), `detail` is only available for `click`, `dblclick`, `mousedown`, and `mouseup` events. I'm guessing that since you can have multiple touches at once, they decided the idea of a "touch sequence" didn't make sense.

Reply:
Makes sense, I'll fix that too. Another nice catch :D